website/contributing/release-stable-patch.md

```diff
@@ -63,7 +67,18 @@ Before continuing further, follow the [testing guide](/contributing/release-test
 
 <AsyncTestingNote/>
 
-### 4. Run `trigger-react-native-release` script
+### 4. (0.74 and later only) Bump monorepo packages
+
+- Version the monorepo package dependencies by running `yarn bump-all-updated-packages`.
+- Push the newly generated commit. A CI workflow will publish the new versions of each monorepo package (excluding `react-native`) to npm.
+- Wait for this workflow to successfully complete before continuing.
+
+```sh
+yarn bump-all-updated-packages # All the package bumps must be on the patch level
+git push origin 0.XX-stable
+```
```

Comment:
Should we copy and paste this command also in the `website/contributing/release-branch-cut-and-rc0.md` file?